.buildkite/x-pack/pipeline.xpack.agentbeat.yml

```diff
@@ -89,6 +89,34 @@ steps:
           - github_commit_status:
               context: "agentbeat: Packaging"
 
+      - label: ":linux: Agentbeat packaging Linux FIPS"
+        key: "agentbeat-package-linux-fips"
+        depends_on:
+          - mandatory-win-2022-system-tests
+        env:
+          PLATFORMS: "+all linux/amd64 linux/arm64"
```

Comment:
QQ: should we really package agentbeat `linux/arm64` on an and64 machine?

Reply:
Agentbeat is packaged for both linux/amd64 and linux/arm64 in the packaging pipeline, so I wanted to check that building for linux/arm64 would still work on PRs and such.
QEMU emulation for golang-crossbuild arm would still work for amd64(host) -> arm64 (image) but if you prefer I can split `linux/arm64` build to a separate `linux/arm64` agent.

Reply:
Separated the amd64 and arm64 builds in 11f4c4994a513ade660c32364692a94d11a40726 and 394149f7e2f796040aef963390df1d7a437c5f21